Document/0x04b-Mobile-App-Security-Testing.md

```diff
@@ -315,7 +315,7 @@ The commit phase is designed to provide instant feedback to developers and as su
 
 ### References
 
-- [manoranjan] - P. Manoranjan. Official (ISC)2 Guide to the CSSLP CBK, Second Edition ((ISC)2 Press) 2nd Edition, 2013
+- [paul] - M. Paul. Official (ISC)2 Guide to the CSSLP CBK, Second Edition ((ISC)2 Press) 2nd Edition, 2014
```

Comment:
Please also change the reference to `#manoranjan` above. Also, please remove one of "Second Edition" or "2nd Edition".